cartographer/mapping/2d/grid_2d.h

```diff
@@ -70,7 +78,10 @@ class Grid2D : public GridInterface {
   // Grows the map as necessary to include 'point'. This changes the meaning of
   // these coordinates going forward. This method must be called immediately
   // after 'FinishUpdate', before any calls to 'ApplyLookupTable'.
-  virtual void GrowLimits(const Eigen::Vector2f& point);
+  virtual void GrowLimits(const Eigen::Vector2f& point) {
```

Comment:
Just noticed this is a virtual method. Perhaps inlining it does not make any sense.